docs/DeveloperGuide.md

```diff
@@ -344,58 +359,56 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 3.
 
-**Use case: Listing out information of all residents**
+**Use case: UC3. Listing out information of all residents**
 
 **MSS**
-1. User wants to see the full list of residents in RC4.
-2. User uses the list command.
-3. RC4HDB displays the details of all residents in RC4.
+1. User wants to see the full list of residents in [**RC4**](#glossary).
```

Comment:
Same as above, perhaps we could add something like "User requests for list of residents from RC4HDB" as an additional bullet point.